src/annotations/AnnotationParser.php

```diff
@@ -13,6 +13,10 @@
 
 namespace mindplay\annotations;
 
+if (!AnnotationManager::$traitsSupported) {
+    define('mindplay\\annotations\\T_TRAIT', -2);
```

Comment:
You can safely check if `T_TRAIT` constant isn't defined and if it is the case do `define('T_TRAIT', 369)` (seems that is that constant value in PHP 5.4).